Core/HLE/sceNet.cpp

```diff
@@ -449,22 +548,135 @@ int sceNetInetTerm() {
 	return 0;
 }
 
-static int sceNetApctlInit() {
+static int sceNetApctlInit(int stackSize, int initPriority) {
 	ERROR_LOG(SCENET, "UNIMPL sceNetApctlInit()");
 	if (netApctlInited)
 		return ERROR_NET_APCTL_ALREADY_INITIALIZED;
+
+	memset(&netApctlInfo, 0, sizeof(netApctlInfo));
+	// Set default values
+	strncpy((char*)&netApctlInfo.name, "Wifi", sizeof(netApctlInfo.name));
```

Comment:
In Common/StringUtil.h we have truncate_cpy which does the right thing with zero-termination, unlike strncpy.

Reply:
hmm.. i can't find Common/StringUtil.h , may be you meant "base/stringutil.h"?